components/vf-form/vf-form__core/assets/vf-form__float-labels.js

```diff
@@ -1,127 +0,0 @@
-// vf-form__float-labels
```

Comment:
Rather than delete all the JS, it would be better if we left the function with a console.log or .warn to update your scripts.js rollup. That way a component update won't break the site's JS and alert the dev they need to update.

Reply:
added to this PR now